### PR TITLE
Upload resized video with thumbnail

### PR DESCRIPTION
**Pull request by maxpierini**
New upload_video features
1. resize video (requires `ffmpeg` installed on OS)
2. thumbnail creation (requires ImageMagick `convert` installed on OS)

### CODE REVIEW

**Comment by ohld** [comment]
LGTM. Thanks for the great work!

**Review by ohld — DISMISSED** [review]

**Comment by ohld** [comment]
Questions:
1) why do we need two video examples, not only one?
2) please add instructions how to install ffmpeg and convert libraries in the readme.md (use *.MD extension for files).

**Comment by maxpierini** [comment]
> Questions:
> 
> 1. why do we need two video examples, not only one?
> 2. please add instructions how to install ffmpeg and convert libraries in the readme.md (use *.MD extension for files).

1. My fault... I thought I uploaded one horizontal and one vertical video. I uploaded two horizontal by mistake. I'll fix. I uploaded two videos also to test random upload feature. Is that ok? Or will this increase too much the repo size?
2. Ok :-)

File: examples/videos/README.md [hunk 1]
```diff
@@ -4,7 +4,9 @@ Upload videos. Thumbnails will be created from instabot.
 ## Requirements
 To resize/crop videos and create thumbnails, are required on your OS
 - `ffprobe` and `ffmpeg`. Tested with version `3.4.4`
+  - Read here how to install `ffmpeg` https://github.com/adaptlearning/adapt_authoring/wiki/Installing-FFmpeg
```
Comment (ohld) [inline examples/videos/README.md:7]: hmmm,  can you please find a proper PIP package to simplify the installation? Being an instabot user it is quite easier to do `pip install python_magic_ffmpeg_package`.

**Review by ohld — CHANGES_REQUESTED** [review]
Last questions:
1) will this all works on windows?
2) can you please find the correspondent python packages that have the same functionallity as `ffmpeg` and `convert`? It is much easier for a user to just `pip install`something instead of installing of heavyweight external CLIs.

File: examples/videos/README.md (hunk 1 above)
Comment (maxpierini) [reply examples/videos/README.md:7]: Closing pull request. New pull request from @maxdevblock using python modules only.

**Comment by maxpierini** [comment]
Closing request. New pull request from @maxdevblock using python modules only.

**Comment by ohld** [comment]
Thanks! 
Great job and thanks for contributing. @maxpierini

**Pull request closed without merging**